snappl/psf.py

```diff
@@ -1124,9 +1124,18 @@ def get_stamp( self, x=None, y=None, x0=None, y0=None, flux=1., seed=None ):
 
             # Note the +1s in galsim.PositionD below; galsim uses 1-indexed pixel positions,
             # whereas snappl uses 0-indexed pixel positions
-            point.drawImage( rmutils.bpass, method='phot', rng=rmutils.rng, photon_ops=photon_ops,
-                             n_photons=self.n_photons, maxN=self.n_photons, poisson_flux=False,
-                             center=galsim.PositionD(stampx+1, stampy+1), use_true_center=True, image=stamp )
+            if self.include_photonOps:
+                point.drawImage( rmutils.bpass, method='phot', rng=rmutils.rng, photon_ops=photon_ops,
+                              n_photons=self.n_photons, maxN=self.n_photons, poisson_flux=False,
+                              center=galsim.PositionD(stampx+1, stampy+1), use_true_center=True, image=stamp )
```

Comment:
Could consider putting a `center=galsim.PositionD(stampx+1, stampy+1)` call immediately after the comment explaining the +1 and immediately before the if/else block.  Then pass `center=center` in the kwargs.